diff -Naur pnm2ppa-1.12/Makefile pnm2ppa-1.12.oden/Makefile
--- pnm2ppa-1.12/Makefile	2008-12-25 21:16:01.000000000 +0100
+++ pnm2ppa-1.12.oden/Makefile	2008-12-25 21:16:38.000000000 +0100
@@ -13,7 +13,8 @@
 #compiler and linker options:
 CC=gcc
 CFLAGS=$(RPM_OPT_FLAGS)
-LDFLAGS=-lm
+LIBS=-lm
+LDFLAGS=
 
 INSTALL=install
 # Install under Linux and other versions with the -b (backup) option
@@ -126,10 +127,10 @@
 	rm -f $(DESTDIR)$(CONFDIR)/pnm2ppa.conf
 
 pnm2ppa: $(OBJS) 
-	$(CC) $(LDFLAGS) $(CFLAGS) -o pnm2ppa $(OBJS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o pnm2ppa $(OBJS) $(LIBS)
 
 calibrate_ppa: calibrate_ppa.o pnmtools.o gamma.o
-	$(CC) $(LDFLAGS) $(CFLAGS) -o calibrate_ppa calibrate_ppa.o pnmtools.o gamma.o 
+	$(CC) $(CFLAGS) $(LDFLAGS) -o calibrate_ppa calibrate_ppa.o pnmtools.o gamma.o $(LIBS)  
 
 calibrate_ppa.o: calibrate_ppa.c global.h image.h defaults.h pnmtools.h
 	$(CC) $(CFLAGS) -c calibrate_ppa.c
diff -Naur pnm2ppa-1.12/pbm2ppa-0.8.6/Makefile pnm2ppa-1.12.oden/pbm2ppa-0.8.6/Makefile
--- pnm2ppa-1.12/pbm2ppa-0.8.6/Makefile	2008-12-25 21:16:01.000000000 +0100
+++ pnm2ppa-1.12.oden/pbm2ppa-0.8.6/Makefile	2008-12-25 21:17:11.000000000 +0100
@@ -1,5 +1,6 @@
 CC=gcc
 CFLAGS=$(RPM_OPT_FLAGS)
+LIBS=-lm
 LDFLAGS=
 PRINTER=HP720
 
@@ -20,10 +21,10 @@
 	rm -f *.o *~ pbmtpg pbm2ppa 
 
 pbm2ppa: pbm2ppa.o ppa.o pbm.o cutswath.o
-	$(CC) $(LDFLAGS) -o pbm2ppa pbm2ppa.o ppa.o pbm.o cutswath.o
+	$(CC) $(CFLAGS) $(LDFLAGS) -o pbm2ppa pbm2ppa.o ppa.o pbm.o cutswath.o
 
 pbmtpg: pbmtpg.o
-	$(CC) $(CFLAGS) -o pbmtpg pbmtpg.o -lm
+	$(CC) $(CFLAGS) $(LDFLAGS) -o pbmtpg pbmtpg.o $(LIBS)
 
 .c.o: $*.c ppa.h pbm.h
 	$(CC) $(CFLAGS) -DDEFAULT_PRINTER=$(PRINTER)  -c $*.c
